learn-resources/data/object-web.json

```diff
@@ -11,6 +11,12 @@
 			"url": "https://learn.liferay.com/en/w/dxp/liferay-development/objects/creating-and-managing-objects/validations/expression-builder-validations-reference"
 		}
 	},
+	"managing-data-from-external-systems": {
+		"en_US": {
+			"message": "Learn more.",
+			"url": "https://learn.liferay.com/w/dxp/liferay-development/objects/integrating-objects-with-third-party-services/managing-data-from-external-systems#proxy-object-limitations"
```

Comment:
Our goal is to send the user to the #proxy-object-limitations section of the documentation, right @DavysonMelo?

Reply:
Yes